pynest/examples/sudoku/helpers.py

```diff
@@ -233,73 +206,113 @@ def plot_field(puzzle, solution, with_color=False):
         puzzle (np.array): array of shape (9,9) that represents the puzzle
         that is being solved. See get_puzzle()
         solution (np.array): array of shape (9,9) representing the solution.
+        ax (plt.Axes): Axes object on which to draw the field.
         with_color (bool, optional): if True, green and red are used to
         indicate which parts of the solution are valid and which are not.
         Otherwise, only black and white are used. Defaults to False.
```

Comment:
Can you please re-format the docstrings to follow the (NumPy documentation format](https://numpydoc.readthedocs.io/en/latest/format.html)? We're using that in all of PyNEST and I think the readybility of this block would drastically benefit from some headlines and indentation.